Tooling-Landscape/Unanimous-Understanding/OSS-Tooling-Landscape-Glossary.md

```diff
@@ -3,46 +3,66 @@
 
 ### Security Scanning
 Dedicated tools which scan source code and/or binaries to detect code vulnerabilities, both in proprietary as well as in FOSS components.
 
-###Product and Component Catalogue
-Software products and most software components are built from other software components. Therefore, a software product can be viewed as a graph of interconnected software components and fragments. The properties of the constituent parts of a product and their relationships are all relevant to product governance and need to be modeled precisely. The product and component catalog contains such information and potentially other meta data.
+Comment Lars: This is a general quality issue, not an Open Source issue, from my point of view. Part of compliance tooling should be the identification of components for which CVEs are publicly available, so that a build can give appropriate feedback. This Security Scanning here is a standard static code analysis and therefore out of scope.
+
+### Product and Component Catalog
+Software products and most software components are built from other software components. Therefore, a software product can be viewed as a graph of interconnected software components and fragments. The Product Catalog contains the information on this graph to enable the management of dependencies over the life cycle of the software, e.g., to ensure that a new vulnerability in a FOSS component can be evaluated and actions toward updating software in the field can be initiated.
+
+### Component Catalog
+A system or service that stores metadata to used 3rd party, especially FOSS components. This includes licenses, copyrights, known vulnerabilitites and ECCN classifications. The information is used within the compliance tooling to aquire the metadata for the identified components, to assess the compilation of dependent components and to provide the information for the creation of the FOSS bundle (i.e., the artifacts needed for the distribution of a software). The Component Catalog can be linked to an external FOSS Metadatabase to retrieve commonly known information and make it usable within the organization. Also the Security Vulnerability Database and other sources for e.g. ECCN classifications are linked to retrieve the necessary information and to make it available within the company.
 
-### ECCN Classifications
+#### ECCN Classifications
 The Export Control Classification Number (ECCN) is an alpha-numeric code that identifies the level of export control for articles, technology and software that are exported from the United States or the European Union. The code is used for any kind of goods, incl. software.
```

Comment:
I faintly remember last meeting said that there should be either "database" or "repo" or "service" to such things. I guess repo would be the preference?